event/views.py

```diff
@@ -65,19 +65,22 @@ def view_event(request, user_id):
 
 def event_list(request, user_id):
     events = Event.manager.all()
-    if request.method == "GET":
-        if 'Choose_filter' in request.GET:
-            filter_type = request.GET.get('Choose_filter')
 
-            if filter_type == 'Category' and 'Choose_Category' in request.GET:
-                category = request.GET.get('Choose_Category')
-                events = events.filter(category__name=category)
-            elif filter_type == 'Location' and 'Choose_Location' in request.GET:
-                location = request.GET.get('Choose_Location')
-                events = events.filter(location__name=location)
+    if 'Choose_Category' in request.GET and request.GET.get('Choose_Category') != 'None':
```

Comment:
You should change this to the same way you check `order_by`
```suggestion
    category = request.GET.get('Choose_Category', 'None')
    if category  != 'None':
```